@kiva/kv-components/vue/KvCommentsAdd.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+	<div class="tw-flex tw-flex-col">
+		<div class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center tw-gap-0.5 md:tw-gap-1">
+			<div class="tw-flex tw-items-center tw-gap-1">
+				<img
+					v-if="userImageUrl"
+					:src="userImageUrl"
+					alt="Lender"
+					class="
+						data-hj-suppress
+						tw-inline-block
+						tw-w-3.5
+						tw-h-3.5
+						tw-rounded-full
+						tw-overflow-hidden
+						tw-object-fill
+					"
+				>
+				<div class="tw-font-medium">
+					{{ userDisplayName }}
+				</div>
+			</div>
+			<kv-text-input
+				:id="ADD_COMMENT_ID"
+				v-model="addCommentValue"
+				placeholder="Add a comment to this loan..."
+				class="tw-grow"
+			/>
+		</div>
+		<div class="tw-flex tw-py-0.5 tw-gap-0.5">
+			<kv-button
+				variant="ghost"
+				class="tw-ml-auto"
+				@click="cancel"
+			>
+				Cancel
+			</kv-button>
+			<kv-button
+				variant="ghost"
+				:state="commentButtonState"
+				@click="comment"
+			>
+				Comment
+			</kv-button>
+		</div>
+	</div>
+</template>
+
+<script>
+import { computed, ref } from 'vue-demi';
+import KvButton from './KvButton.vue';
+import KvTextInput from './KvTextInput.vue';
+
+export const ADD_COMMENT_ID = 'add-comment-value';
+export const ADD_COMMENT_EVENT = 'add-comment';
+
+export default {
+	components: {
+		KvButton,
+		KvTextInput,
+	},
+	props: {
+		/**
+		 * The full URL for the user image
+		 */
+		userImageUrl: {
+			type: String,
+			default: '',
+		},
+		/**
+		 * The name to display for the user
+		 */
+		userDisplayName: {
+			type: String,
+			default: '',
+		},
+	},
+	emits: [ADD_COMMENT_EVENT],
+	setup(_props, { emit }) {
+		const addCommentValue = ref('');
+
+		const commentButtonState = computed(() => (addCommentValue.value ? '' : 'disabled'));
+
+		const cancel = () => {
+			addCommentValue.value = '';
+		};
+
+		const comment = () => {
+			emit(ADD_COMMENT_EVENT, addCommentValue.value);
+		};
+
+		return {
+			ADD_COMMENT_ID,
+			addCommentValue,
+			commentButtonState,
+			cancel,
+			comment,
+		};
+	},
+};
+</script>
+
+<style lang="postcss" scoped>
+>>> input {
```

Comment:
Can we use the deep() syntax here instead?

Reply:
Nope, not supported currently in this project (not true Vue3).